Resources/Audio/_Nuclear14/Weapons/Firing/attributions.yml

```
@@ -0,0 +1,4 @@
+﻿- files: ["war_never_changes.ogg"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "BigIron github and it's sources"
```

```suggestion
  copyright: "BigIron github and its sources"
```
This is quite an unclear Copyright. You should clarify what 'BigIron' is (I.E, a repo, a person, etc.) and what 'its sources' if referring to, and potentially include an actual link somewhere if applicable.

---

The copyright is a local open sourced github that closed down. I may aswell not put any copyright there.

---

That's not how copyright works.